src/test/cpp/gtester/AliasTest.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <vector>
+#include <gtest/gtest.h>
+#include <alias/Alias.h>
+
+using namespace stride::alias;
+
+namespace Tests {
+
+TEST(AliasDeathTest, Alias)
+{	
+	//Ruzie met death
```

Comment:
This might be because calls to `assert` are elided when the application is compiled in release mode, which is like, always, I guess.